pom.xml

```diff
@@ -25,7 +25,7 @@
     </parent>
     <groupId>org.kill-bill.billing.plugin</groupId>
     <artifactId>killbill-plugin-api</artifactId>
-    <version>0.10-SNAPSHOT</version>
+    <version>0.11-SNAPSHOT</version>
```

Comment:
I don't know what I should be making the version, but since I am working locally on my machine, I assume this will be fine for now.